extensions/wikia/ArticleVideo/ArticleVideoController.class.php

```diff
@@ -12,9 +12,15 @@ public function featured() {
 
 			$this->setVal( 'videoDetails', $featuredVideoData );
 			if ( ArticleVideoContext::isJWPlayer( $featuredVideoData ) ) {
-                $this->setVal( 'jwplayerInstance', file_get_contents($this->getApp()->wg->extensionsPath . '/wikia/ArticleVideo/scripts/featured-video.jwplayer.instance.js') );
-				$this->setVal( 'featuredVideoAutoplay', file_get_contents($this->getApp()->wg->extensionsPath . '/wikia/ArticleVideo/scripts/featured-video.autoplay.js') );
-				$this->setVal( 'jwplayerScript', file_get_contents($this->getApp()->wg->extensionsPath . '/wikia/ArticleVideo/scripts/featured-video.jwplayer.instant.js') );
+                $jwPlayerScript = $this->wg->out->getResourceLoader()->getModule( 'ext.ArticleVideo.jw' )->getScript(
+					new ResourceLoaderContext( new ResourceLoader(), $this->wg->request)
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Refactor this code to rely on RequestContext instead of context-dependent globals. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=wikia-php-rules%3AContextualWgVariableUsage)